client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -28,13 +28,28 @@
     <!-- title column -->
     <ng-container matColumnDef="title">
         <mat-header-cell *matHeaderCellDef mat-sort-header>Topic</mat-header-cell>
-        <mat-cell *matCellDef="let item">{{ item.getListTitle() }}</mat-cell>
+        <mat-cell (click)="onTitleColumn(item)" *matCellDef="let item">{{ item.getListTitle() }}</mat-cell>
     </ng-container>
 
-    <!-- Duration column -->
-    <ng-container matColumnDef="duration">
-        <mat-header-cell *matHeaderCellDef mat-sort-header>Duration</mat-header-cell>
-        <mat-cell *matCellDef="let item">{{ item.duration }}</mat-cell>
+    <!-- Info column -->
+    <ng-container matColumnDef="info">
+        <mat-header-cell *matHeaderCellDef mat-sort-header>Info</mat-header-cell>
+        <mat-cell (click)="onInfoColumn(item)" *matCellDef="let item">
+            <div class="info-col-items">
+                <div *ngIf="item.visibility">
+                    <mat-icon>visibility</mat-icon>
+                    {{ item.visibility | translate }}
```

Comment:
Is this handled properly by the translation service, so that all possible strings appear?

Reply:
yes